packages/app/ui/buttons/BuyNowBtn.tsx

```diff
@@ -37,29 +37,34 @@ export function BuyNowBtn({ entry }: Props) {
         }}
         useTouchable
       />
-      <PaymentConfirmationModal
-        visible={modalVisible}
-        entry={entry}
-        price={price.price}
-        initialEquityForSale={price.amount}
-        hideModal={(success: boolean) => {
-          setModalVisible(false);
-          if (success) {
-            mutate(getEntryOfferUrl(entry.code!, entry.issuer!));
+      <ApolloConsumer>
+        {(client) => (
```

Comment:
we are already using `useApolloClient` in this component, so I don't think it's necessary, you can use that hook.